billy_metadata/ar.py

```diff
@@ -0,0 +1,193 @@
+            'start_date': datetime.date(2015, 5, 26),
+            'end_date': datetime.date(2015, 5, 28),
+            'display_name': '2015 First Extraordinary Session',
+            '_scraped_name': 'First Extraordinary Session, 2015',
+            'slug': '2015S1',
+        },
+        '2016S2': {
+            'type': 'special',
+            'start_date': datetime.date(2016, 4, 6),
+            'end_date': datetime.date(2016, 4, 6),
+            'display_name': '2016 Second Extraordinary Session',
+            '_scraped_name': 'Second Extraordinary Session, 2016',
+            'slug': '2016S2',
+        },
+        '2016F': {
+            'type': 'special',
+            'start_date': datetime.date(2016, 4, 13),
+            'end_date': datetime.date(2016, 5, 9),
+            'display_name': '2016 Fiscal Session',
+            '_scraped_name': 'Fiscal Session, 2016',
+            'slug': '2016F',
+        },
+        '2016S3': {
+            'type': 'special',
+            'start_date': datetime.date(2016, 5, 19),
+            'display_name': '2016 Third Extraordinary Session',
+            '_scraped_name': 'Third Extraordinary Session, 2016',
+            'slug': '2016S3',
+        },
+        '2017': {
+            'type': 'primary',
+            'start_date': datetime.date(2017, 1, 12),
+            'end_date': datetime.date(2017, 4, 22),
+            'display_name': '2017 Regular Session',
+            '_scraped_name': 'Regular Session, 2017',
+            'slug': '2017R',
+        },
+    },
+    'feature_flags': ['influenceexplorer', 'events'],
+    '_ignored_scraped_sessions': [
+        'Regular Session, 2009',
+        'Fiscal Session, 2010',
+        'Regular Session, 2007',
+        'First Extraordinary Session, 2008',
+        'Regular Session, 2005',
+        'First Extraordinary Session, 2006 ',
+        'Regular Session, 2003 ',
+        'First Extraordinary Session, 2003',
+        'Second Extraordinary Session, 2003',
+        'Regular Session, 2001 ',
+        'First Extraordinary Session, 2002',
+        'Regular Session, 1999',
+        'First Extraordinary Session, 2000',
+        'Second Extraordinary Session, 2000',
+        'Regular Session, 1997 ',
+        'Regular Session, 1995 ',
+        'First Extraordinary Session, 1995 ',
+        'Regular Session, 1993 ',
+        'First Extraordinary Session, 1993 ',
+        'Second Extraordinary Session, 1993',
+        'Regular Session, 1991',
+        'First Extraordinary Session, 1991 ',
+        'Second Extraordinary Session, 1991 ',
+        'Regular Session, 1989',
+        'First Extraordinary Session, 1989',
+        'Second Extraordinary Session, 1989',
+        'Third Extraordinary Session, 1989 ',
+        'Regular Session, 1987 ',
+        'First Extraordinary Session, 1987',
+        'Second Extraordinary Session, 1987',
+        'Third Extraordinary Session, 1987',
+        'Fourth Extraordinary Session, 1987',
+    ],
+}
+
+
+def session_list():
```

Comment:
Could you move this function to a method of the `Arkansas` class and rename it to `get_session_list`?